continuationsPlugin/src/test/resources/StateMachineTwoContinuationsChainedTwoValPriorTwoValBetweenTwoValAfterChainIgnored.scala

```diff
@@ -92,20 +85,11 @@ package continuations {
                       return[label1] ()
                   }
               case 1 =>
-                y##1 = $continuation.I$0
```

Comment:
how do we track the different values before and after `case 1` then? if it returns before case 1 and then comes back later on to continue from here